### ISSUE TITLE
[no-unused-vars] false positive with import used in type

### ISSUE THREAD

**Issue by wesleymostien** [bradzacher/eslint-plugin-typescript#90]
![screen shot 2017-11-05 at 11 53 30](https://user-images.githubusercontent.com/6223285/32414104-fb49889e-c21f-11e7-9faa-79ad82d36031.png)

As you can see in the screenshot, eslint reports an error on 'webpack' while it is used in a type notation.

**Comment by macklinu** [comment]
I added the following test locally as a valid test case, and all tests passed:

```diff
diff --git a/tests/lib/rules/no-unused-vars.js b/tests/lib/rules/no-unused-vars.js
index c470d3c..0bfc682 100644
--- a/tests/lib/rules/no-unused-vars.js
+++ b/tests/lib/rules/no-unused-vars.js
@@ -316,6 +316,13 @@ import { Another } from 'some';
 interface A extends Nullable<SomeOther> {
     do(a: Nullable<Another>);
 }
+        `,
+        `
+import * as webpack from 'webpack';
+
+const webpackConfig: webpack.Configuration = {};
+
+return webpackConfig;
         `
     ],
 
```

Nothing in that snippet is being reported as unused. Are you still experiencing this issue with the latest version of eslint-plugin-typescript, and if so, could you please include your ESLint config?

**Comment by bradzacher** [comment]
closing as we are unable to reproduce, and there has been no response from OP.

**Comment by bradzacher** [comment]
Check your parser version.
Plugin version 0.14.0 only supports up to parser v17 (i think it was 17?)

**Comment by JPeer264** [comment]
Oh. I am currently on `v21`. So with the plugin version `v1` the current parser should be supported right?

**Comment by bradzacher** [comment]
Yes. With the v1 release we have made it possible for us to control the parser version to help prevent mismatches.
And we have upgraded and handled the changes introduced in parser v21.

We're close to the v1 release!

**Comment by eddiemonge** [comment]
How close?
